File: calculator.java
import java.util.Scanner;

public class calculator {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		System.out.print("Enter the first number: ");
		double firstNumber = scanner.nextDouble();

		System.out.print("Enter an operator (+, -, *, /): ");
		String operator = scanner.next();

		System.out.print("Enter the second number: ");
		double secondNumber = scanner.nextDouble();

		double result;

		switch (operator) {
			case "+":
				result = firstNumber + secondNumber;
				break;
			case "-":
				result = firstNumber - secondNumber;
				break;
			case "*":
				result = firstNumber * secondNumber;
				break;
			case "/":
				if (secondNumber == 0) {
					System.out.println("Error: Cannot divide by zero.");
					scanner.close();
					return;
				}
				result = firstNumber / secondNumber;
				break;
			default:
				System.out.println("Error: Invalid operator. Use +, -, *, or /.");
				scanner.close();
				return;
		}

		System.out.println("Result: " + result);
		scanner.close();
	}
}

File: Calculator.py
def calculate(first, operator, second):
	if operator == "+":
		return first + second
	if operator == "-":
		return first - second
	if operator == "*":
		return first * second
	if operator == "/":
		if second == 0:
			raise ZeroDivisionError
		return first / second
	raise ValueError("Invalid operator")


while True:
	operator = input("Enter an operation (+, -, *, /) or q to quit: ").strip().lower()
	if operator in ("q", "quit"):
		print("Goodbye!")
		break

	if operator not in ("+", "-", "*", "/"):
		print("Invalid operator.")
		continue

	try:
		first = float(input("Enter the first number: "))
		second = float(input("Enter the second number: "))
		print(f"Result: {calculate(first, operator, second)}")
	except ValueError as error:
		if str(error) == "Invalid operator":
			print("Invalid operator.")
		else:
			print("Invalid number.")
	except ZeroDivisionError:
		print("Cannot divide by zero.")
